__tests__/app.test.js

```diff
@@ -119,10 +120,58 @@ describe("3 GET users /api/users", () => {
   });
   it("status 404, should ", () => {
     return request(app)
-      .get("/api/banana")
+      .get("/api/apple")
       .expect(404)
+  });
+});
+
+describe('6 PATCH /api/reviews/:review_id', () => {
+  it('status 200, should add a number of votes to the selected review', () => {
+    return request(app)
+      .patch("/api/reviews/1")
+      .expect(200)
+      .send({ inc_votes: 3})
       .then(({ body }) => {
-        expect(body).toEqual({});
+        expect(body.review[0]).toEqual({
+          review_id: 1,
+          title: 'Agricola',
+          category: 'euro game',
+          designer: 'Uwe Rosenberg',
+          owner: 'mallionaire',
+          review_body: 'Farmyard fun!',
+          review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+          created_at: '2021-01-18T10:00:20.514Z',
+          votes: 4
+        });
+      });
+  });
+  it("status 404, should return an error", () => {
+    const review_id = 999;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "No review found for review_id:999" });
+      });
+  });
+  it("status 400, should return an error", () => {
+    const review_id = "pear";
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "invalid type" });
       });
```

Comment:
Need some more test - 
400  send({})
400 send({ inc_votes: "banana"})

Reply:
Sorry Jon. I didn't see the extra tests  - it's about the patching body. Both 400. 
return request(app)
      .patch(`/api/reviews/${review_id}`)
     **_.send({inc_votes: "banana})_**
     .expect(404)
      .then(({ body }) => {
        expect(body.msg).toBe(".......")
      });
  });